docs/operations.md

```diff
@@ -0,0 +1,132 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+### Compatibility
+
+- JDK17
+- Operator used fabric8 which assumes to be compatible with available k8s versions. However for using status subresource, please use k8s version 1.14 or above.
```

Comment:
As I pinged you already, the K8s eco-system is moving faster in the public environment. 
- https://github.com/apache/spark/pull/47990
- https://github.com/apache/spark-kubernetes-operator/pull/117

Just FYI, in the community, please don't claim which you didn't test at all. AFAIK, nobody tests K8s 1.14 in these days.